README.md

```diff
@@ -3,7 +3,7 @@
 ---
 
 <p align="center">
-  <img src="docs/public/svg/favicon.svg" style="width:12vw" alt="Amplify UI logo" />
+  <img src="docs/public/svg/favicon.svg" style="width:12vw" alt="Amplify UI Logo" />
```

Comment:
Info: why should "logo" be capitalized?

Reply:
this was just a test run of the CI/CD, that's why it's labeled DO NOT MERGE :)

Reply:
Ah, makes sense!